.github/workflows/pages.yaml

```diff
@@ -0,0 +1,24 @@
+name: Update Cumulus docs
+on:
+  push:
+    branches: ["main"]
+    paths: ["docs/**"]
+
+jobs:
+  update-docs:
+    name: Update Cumulus docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send workflow dispatch
+        uses: actions/github-script@v6
```

Comment:
v7 is latest if you care :shrug: - every single action in the universe upgraded versions a while back to rebase on node 20

Reply:
I'll change this here and also in the next library PR.